Running test on mutant:
============================= test session starts ==============================
platform linux -- Python 3.8.19, pytest-8.3.3, pluggy-1.5.0
rootdir: /mnt/temp/inspect_mutant
collected 1 item

../test.py F

=================================== FAILURES ===================================
_____________________________________ test _____________________________________

    def test():
>       assert _multiline_with_doc.display_index == 2
E       assert 3 == 2
E        +  where 3 = _AttributeGroupFormatter(display_index=3, formatter=<function _format_multiline_with_doc at 0x7c73ee683820>).display_index

../test.py:5: AssertionError
=========================== short test summary info ============================
FAILED ../test.py::test - assert 3 == 2
============================== 1 failed in 0.03s ===============================
